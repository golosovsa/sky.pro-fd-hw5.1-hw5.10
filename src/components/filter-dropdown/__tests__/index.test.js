import ShallowRenderer from 'react-test-renderer/shallow'
import FilterDropdown from '..'

describe('<FilterDropdown />', () => {
  describe('Snapshot test', () => {
    it('should render successfully and match snapshot', () => {
      const renderer = new ShallowRenderer()
      const tree = renderer.render(<FilterDropdown>123</FilterDropdown>)

      expect(tree).toMatchSnapshot()
    })
  })

  describe('Snapshot test with tab and button', () => {
    it('should render successfully and match snapshot', () => {
      const renderer = new ShallowRenderer()
      const tree = renderer.render(
        <FilterDropdown button={123} tab={123} isShowPopup={true}>
          123
        </FilterDropdown>
      )

      expect(tree).toMatchSnapshot()
    })
  })
})
