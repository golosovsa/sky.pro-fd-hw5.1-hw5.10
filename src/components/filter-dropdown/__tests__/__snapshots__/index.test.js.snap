// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FilterDropdown /> Snapshot test should render successfully and match snapshot 1`] = `
<styled.div
  mouseEventsDisabled={false}
>
  <styled.div>
    <styled.div>
      123
    </styled.div>
  </styled.div>
</styled.div>
`;

exports[`<FilterDropdown /> Snapshot test with tab and button should render successfully and match snapshot 1`] = `
<styled.div
  mouseEventsDisabled={false}
>
  123
  <styled.div>
    <styled.div>
      123
    </styled.div>
  </styled.div>
  <styled.div>
    <styled.p>
      123
    </styled.p>
  </styled.div>
</styled.div>
`;
